lib/ansible/modules/cloud/azure/azure_rm_acs.py

```diff
@@ -325,8 +330,10 @@ def create_master_profile_instance(masterprofile):
     '''
     return ContainerServiceMasterProfile(
         count=masterprofile[0]['count'],
-        dns_prefix=masterprofile[0]['dns_prefix']
-    )
+        dns_prefix=masterprofile[0]['dns_prefix'],
+        vm_size=masterprofile[0]['vm_size'],
+        first_consecutive_static_ip=None
```

Comment:
So far, Azure server doesn't accept request with body that contains `first_consecutive_static_ip` property.

Reply:
should add some comments to explain why we always want to set it to none.